numba/cuda/cudaimpl.py

```diff
@@ -129,8 +129,8 @@ def ptx_cmem_arylike(context, builder, sig, args):
     kstrides = [context.get_constant(types.intp, s) for s in arr.strides]
     context.populate_array(ary,
                            data=builder.bitcast(genptr, ary.data.type),
-                           shape=cgutils.pack_array(builder, kshape),
-                           strides=cgutils.pack_array(builder, kstrides),
+                           shape=cgutils.pack_array(builder, kshape, types.intp),
+                           strides=cgutils.pack_array(builder, kstrides, types.intp),
```

Comment:
This line is too long - how would you like it indented?

Perhaps the following?
```
context.populate_array(
     ary,
     data=builder.bitcast(genptr, ary.data.type),
     # etc, with less wasted space on the left
)
```